c_examples/regs-dump.c

```diff
@@ -14,17 +14,17 @@ void read_registers(void* driver, const char* vm_name) {
     Registers regs;
     memset(&regs, 0, sizeof(regs));
     if (microvmi_read_registers(driver, 0, &regs)) {
-        printf("rax: 0x%" PRIx64 "\n", regs.x86._0.rax);
```

Comment:
How could this even happen? Shouldn't this have been cought by the CI when the enum was removed?

Reply:
I believe this happened because we install cbindgen from the latest version, and the last one broke this enum, so I had to fix it.
best way to fix this is to pin cbindgen crate both in CI and c_examples/README.md

Reply:
Ok, I see. I would actually prefer not to pin the version. Unless of course it becomes too much of a hassle to maintain compatibility withh newer releases of cbindgen.

Reply:
I mean, this would have been caught by our CI if we would run it periodically. Also, I think the result is an improvement (although just a small one).

Reply:
> I would actually prefer not to pin the version. 

If we don't pin, we let our users try our library and eventually fail to compile the `c_examples`, unbeknownst to us.

> I mean, this would have been caught by our CI if we would run it periodically.

Running the CI periodically is a solution to the wrong problem.
we need to ping cbindgen, and periodically update it